src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1527,8 +1527,7 @@ protected void VisitStatementsWithLocalFunctions(BoundBlock block)
                 {
                     if (stmt.Kind == BoundKind.LocalFunctionStatement)
                     {
-                        VisitLocalFunctionStatement(
-                            (BoundLocalFunctionStatement)stmt);
+                        VisitAlways(stmt);
```

Comment:
Why was this changed? Same with accessibility? I assume just for style/code cleanup, but I just want to be sure.

Reply:
Actually this is semantically important. VisitAlways marks the entry and exit for regions. When just visiting we skipped over the entry and exit, so statements were being assigned to the wrong regions.